### PR TITLE
Testpy3

### PR DESCRIPTION
**Pull request by ShangYuChiang**
Update  files as below:
.travis.yml
gff3tool/bin/gff3_merge.py
gff3tool/bin/gff3_sort.py

### CODE REVIEW

**Comment by mpoelchau** [comment]
I tested gff3_ID_generator in python3 - it works!

File: gff3tool/bin/gff3_to_fasta.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#! /usr/local/bin/python2.7
+#! /usr/local/bin/python3
+from __future__ import print_function
```
Comment (hsiaoyi0504) [inline gff3tool/bin/gff3_to_fasta.py]: It seems to me this line is not necessary.

File: gff3tool/lib/gff3_merge/revision.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python3
 """
 After the revision of gff file done by annotators (spreadsheet), incorporating the revised information into the Web Apollo gff before gff integration. Additionally, if there are features containing 'replace' field at gene level, add 'replace' field to every child level of the gene and then delete the replace field of the gene.
 """
-
+from __future__ import print_function
```
Comment (hsiaoyi0504) [inline gff3tool/lib/gff3_merge/revision.py]: same here.

File: .travis.yml [hunk 3]
```diff
@@ -4,10 +4,11 @@ matrix:
   include:
   - os: linux
     language: python
-    python: 2.7
+    python: 3.7
     install:
-      - pip install .
       - pip install coveralls coverage codecov
+      #- python setup.py build
```
Comment (hsiaoyi0504) [inline .travis.yml]: This line can be removed.

File: .travis.yml [hunk 4]
```diff
@@ -18,22 +19,14 @@ matrix:
       - coverage run -a ./gff3tool/bin/gff3_merge.py -g1 example_file/new_models_w_replace.gff3 -g2 example_file/reference.gff3 -f example_file/reference.fa -og merged.gff -r merged_report.txt -noAuto
       - coverage run -a ./gff3tool/bin/gff3_sort.py -g example_file/example.gff3 -og example-sorted.gff3
       - coverage run -a ./gff3tool/bin/gff3_to_fasta.py -g example_file/example.gff3 -f example_file/reference.fa -st all -d simple -o test_sequences
-
-  - os: linux
-    language: python
-    python: 2.7
-    install:
-      - pip install --no-cache-dir gff3tool
```
Comment (hsiaoyi0504) [inline .travis.yml]: This should not be removed. It's for testing installation package directly from PyPI.

File: .travis.yml [hunk 5]
```diff
@@ -18,22 +19,14 @@ matrix:
...
-  - os: osx
-    language: generic
-    install:
-      - pip2 install .
   - os: osx
     language: generic
+    python: 3.7
     install:
-      - pip2 install --no-cache-dir gff3tool
```
Comment (hsiaoyi0504) [inline .travis.yml]: Same here.

File: gff3tool/bin/gff3_to_fasta.py (hunk 1 above)
Comment (ShangYuChiang) [reply gff3tool/bin/gff3_to_fasta.py]: Yes, It's python3.  Python 3 already has the print function, so the from __future__ import print_function will have no necessary. Thank you.

**Pull request closed without merging**